manifests/repo/rhel.pp

```diff
@@ -4,20 +4,32 @@
 class rabbitmq::repo::rhel (
   $location          = "https://packagecloud.io/rabbitmq/rabbitmq-server/el/${facts['os'][release][major]}/\$basearch",
   String $key_source = $rabbitmq::package_gpg_key,
```

Comment:
should we make this less ambiguous to match `$rabbitmq_key_source`, like `$repo_key_source`, which I think is what this key is now?